src/main/resources/templates/apidocumentation/apiDocumentationForTag.ftl

```diff
@@ -1,9 +1,167 @@
 <#include '../layout.ftl'>
+<#include 'apiDocumentationNav.ftl'>
+<#import '../govuk/accordion.ftl' as govukAccordion>
 
-<@defaultPage pageHeading="${tagName}">
-  <#list operationList as operation>
-    <@govukDetails.details summaryTitle="${operation.getSummary()}">
-      ${operation.getDescription()!operation.getSummary()}
-    </@govukDetails.details>
-  </#list>
-</@defaultPage>
\ No newline at end of file
+<@defaultPage pageHeading="" twoThirdsColumn=false robotsMeta="all">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-quarter">
+      <@apiDocumentationNav/>
+    </div>
+    <div class="govuk-grid-column-three-quarters">
+      <span class="govuk-caption-xl">API endpoints</span>
+      <h1 class="govuk-heading-xl">${tagName}</h1>
+      <p class="govuk-body">
+        These endpoints return data in PDF format for energy labels, PNG or JPEG for arrow images and JSON for errors.
+      </p>
+      <p class="govuk-body">
+        You can import <a href="#" class="govuk-link">the energy label API OpenAPI 3 document</a> into API testing tools
+        like Postman to test all of our API endpoints.
+      </p>
+      <p class="govuk-body">
+        The values you send will be validated. If there are any validation errors, the response body will be a JSON object
```

Comment:
I think it would be better to have this in a top level item about validation (i.e in the same section as the rate limits), rather than repeating it on every endpoint, as its always going to be the same and you have to scroll past the json example to get to the actual endpoint specific content. 
@matteason thoughts?